test/test/nativemem/NativememTests.java

```diff
@@ -145,4 +145,13 @@ public void jfrNoLeaks(TestProcess p) throws Exception {
     public void jfrNoFree(TestProcess p) throws Exception {
         assertNoLeaks(p);
     }
+
+    @Test(mainClass = CallsAllNoLeak.class, os = Os.LINUX, args = "once", env = {"LD_PRELOAD=%lib", "ASPROF_COMMAND=start,nativemem,file=%f.jfr"})
+    public void ldpreload(TestProcess p) throws Exception {
+        p.waitForExit();
+        Output out = Output.convertJfrToCollapsed(p.getFilePath("%f"), "--nativemem");
+        assert out.contains("JavaMain");
+        assert out.contains("JVM_");
+        assert out.contains("malloc_hook");
```

Comment:
Could we use `assertNoLeaks(p)` here, since we are using `CallsAllNoLeak`?

Reply:
Sure, added the call.